doc/source/user_guide/ug_case_settings.rst

```diff
@@ -359,7 +359,8 @@ forcing_nml
    "", "``mm_per_sec``", "(same as MKS units)", ""
    "", "``m_per_sec``", "", ""
    "``restore_ocn``", "logical", "restore sst to data", "``.false.``"
-   "``tfrz_option``", "``linear_salt``", "linear functino of salinity (ktherm=1)", "``mushy``"
+   "``tfrz_option``","``constant``", "constant ocean freezing temperature (Tocnfrz)"
+   "", "``linear_salt``", "linear function of salinity (ktherm=1)", "``mushy``", ""
```

Comment:
Is there an extra pair of quotation marks in this line?

Reply:
The "mushy" default value needs to be moved to the "constant" line then I think it's OK.